cedar-policy/src/api.rs

```diff
@@ -3355,7 +3355,7 @@ mod schema_based_parsing_tests {
             .expect_err("should fail due to type mismatch on manager");
         assert!(
             err.to_string()
-                .contains(r#"In attribute "manager" on Employee::"12UA45", expected a literal entity reference, but got "34FB87""#),
```

Comment:
[nit] should "In" be "in"?

Reply:
It seems the uppercase `In` is caused by another type. I'll fixed it in another PR.